starters/lit-mobx-vaadin/src/pages/page.mixin.ts

```diff
@@ -2,19 +2,42 @@ import { LitElement, css } from 'lit';
 
 type Constructor<T> = new (...args: any[]) => T;
 
-export const PageMixin = <T extends Constructor<LitElement>>(superClass: T) => {
-  class PageMixin extends superClass {
+export const PageMixin = <T extends Constructor<LitElement>>(superClass: T) =>
+  class extends superClass {
     static styles = [
       css`
         ul {
-            list-style: none;
+          padding: 0;
+          list-style: none;
         }
-        
+
+        li {
+          margin: 10px 0;
+          font-size: 1.2rem;
+          text-align: center;
+        }
+
         h1 {
-            border-bottom: 5px solid var(--td-primary-color, var(--td-text-color, #1d4ed8));
+          padding: 15px;
+          font-size: 2rem;
+          font-weight: 700;
+          text-align: center;
+          border-bottom: 5px solid rgb(29 78 216);
```

Comment:
shouldn't all this colors become css variables?, i haven't checked all the colors, but if some are too similar, could it be that the reference is a bit inconsistent with coloring?

Reply:
I made a primary color variable and also some link specific ones since they have unique colors separate from any palette. Let me know what you think :)